lib/src/cancelable_operation.dart

```diff
@@ -212,6 +221,15 @@ class CancelableCompleter<T> {
   }
 
   /// Cancel the completer.
```

Comment:
///  Cancels the operation.

Calling `cancel` on the completer ... well, trivially "cancels the completer". That's not saying anything. Logically, it cancels the *operation* that the completer is used to represent.
(And should use active verb.)